client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -34,34 +33,28 @@ class DesignTypeWithStoreStep extends Component {
 
 	getChoices() {
 		const { translate } = this.props;
-
-		if ( abtest( 'signupStepOneCopyChanges' ) === 'modified' ) {
-			// Note: Don't make this translatable because it's only visible to English-language users
-			return [
-				{ type: 'blog',
-					label: 'A blog',
-					description: 'To share your ideas, stories, and photographs with your followers.',
-					image: <BlogImage /> },
-				{ type: 'page',
-					label: 'A website',
-					description: 'To promote your business, organization, or brand and connect with your audience.',
-					image: <PageImage /> },
-				{ type: 'grid',
-					label: 'A portfolio',
-					description: 'To present your creative projects in a visual showcase.',
-					image: <GridImage /> },
-				{ type: 'store',
-					label: 'An online store',
-					description: 'To sell your products or services and accept payments.',
-					image: <StoreImage /> },
-			];
-		}
+		const blogText = translate( 'To share your ideas, stories, and photographs with your followers.' ), // eslint-disable-line max-len
+			siteText = translate( 'To promote your business, organization, or brand and connect with your audience.' ), // eslint-disable-line max-len
+			gridText = translate( 'To present your creative projects in a visual showcase.' ),
+			storeText = translate( 'To sell your products or services and accept payments.' );
 
 		return [
-			{ type: 'blog', label: translate( 'A list of my latest posts' ), image: <BlogImage /> },
-			{ type: 'page', label: translate( 'A welcome page for my site' ), image: <PageImage /> },
-			{ type: 'grid', label: translate( 'A grid of my latest posts' ), image: <GridImage /> },
-			{ type: 'store', label: translate( 'An online store' ), image: <StoreImage /> },
+			{ type: 'blog',
+				label: translate( 'Start with a blog' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Start with Bluehost' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove-signup-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).